extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -1399,7 +1399,7 @@ export class ProjectsController {
 		if (utils.getAzdataApi()) {
 			let createProjectFromDatabaseDialog = this.getCreateProjectFromDatabaseDialog(profile as azdataType.IConnectionProfile);
 
-			createProjectFromDatabaseDialog.createProjectFromDatabaseCallback = async (model) => await this.createProjectFromDatabaseCallback(model);
+			createProjectFromDatabaseDialog.createProjectFromDatabaseCallback = async (model) => await this.createProjectFromDatabaseCallback(model, profile as azdataType.IConnectionProfile);
```

Comment:
in case the user changes the connection after opening the create project from db dialog, the connection profile (or connection id if that's all that's needed) should be passed into this callback when it's called by the dialog

Reply:
Addressed.